src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRootRenderer.cs

```diff
@@ -113,9 +113,12 @@ public override void ViewDidLoad()
 			tracker.IsRootPage = true;
 			tracker.ViewController = this;
 
+			_tracker = tracker;
 			if (ShellSection.CurrentItem != null)
+			{
 				tracker.Page = ((IShellContentController)ShellSection.CurrentItem).GetOrCreateContent();
-			_tracker = tracker;
+				NavigationItem.Title = _shellContext.Shell.Toolbar.Title;
```

Comment:
I think the bug with this one is that Disappearing is never fired on the root page.

Like if you set a break point on disapearing

```C#
	public class Issue20199Page : ContentPage {

		protected override void OnAppearing()
		{
			base.OnAppearing();
		}

		protected override void OnDisappearing()
		{
			base.OnDisappearing();
		}
	}
```

disappearing never fires

if we get disappearing to fire (or perhaps get appearing to not fire)
That should fix the title